### PR TITLE
[DR-2231] Add azure dev environment

### PR DESCRIPTION
**Pull request by samanehsan**
Set up resources for new Azure environment in the dev data repo google project. These changes must be applied manually.

### CODE REVIEW

File: old/dev.tf.ctmpl [hunk 1]
```diff
@@ -158,6 +159,14 @@ locals {
...
       type = "CNAME"
       rrdatas = "jade-global-tc.datarepo-{{$environment}}.broadinstitute.org."
+    },
+    jade-global-az = {
+      type = "A"
+      rrdatas = "${google_compute_global_address.jade-initials-ip["az"].address}"
+    },
+    jade-az = {
+      type = "CNAME"
+      rrdatas = "jade-global-nu.datarepo-{{$environment}}.broadinstitute.org."
```
Comment (nmalfroy) [inline old/dev.tf.ctmpl:169]: should this be: `jade-global-az.datarepo-{{$environment}}.broadinstitute.org.`?

Comment (samanehsan) [reply old/dev.tf.ctmpl:169]: Yes! Thanks for catching that 🤦‍♀️

Comment (nmalfroy) [reply old/dev.tf.ctmpl:169]: hahah, I'm just happy because I feel like you're usually the one who catches those from me :)

**Review by nmalfroy — APPROVED** [review]
lgtm!

**Pull request merged**